src/css/profile/wearable/changeable/theme-circle/theme.changeable.less

```diff
@@ -368,3 +368,8 @@
 @color_button_icon_bg: F022L1i;
 @color_button_icon_active_bg: F022L1iP;
 @color_button_icon_disabled_bg: F022L1iD;
+/***************************************************************************
+						Fast Scroll
+****************************************************************************/
+@color_scrollview_handler_bg: @fast-scroll-bg-color;
+@color_scrollview_handler_thumb: fade(@white-color, 80%);
```

Comment:
shouldn't this be @color_fastscroll_handler_bg ?

Reply:
This is color of arrows on fast scroll hadler.

Reply:
This is proper value.